### PR TITLE
docs: updated instructions for making your own startingpoint fork

### PR DESCRIPTION
**Pull request by Arcitec**
Edit: It's ready for review now. I ended up doing a bunch of cleanups to the flow of related sections too.

### CODE REVIEW

File: docs/making-your-own.md [hunk 1]
```diff
@@ -29,8 +29,9 @@ The instructions below recommend `startingpoint` because it:
 - Is made to be modified and customized
 - Is more suitable for personal usage
 - Has an easier to edit configuration format, YAML, which also supports comments and inline documentation
+- Is very easy to customize in advanced ways via personal scripts, which "magically" run at build by [simply placing them in the appropriate folders](https://github.com/ublue-os/startingpoint/blob/template/scripts/README.md)
```
Comment (castrojo) [inline docs/making-your-own.md:32]:
```suggestion
- Advanced customization can be accomplished at built time by [placing scripts in the appropriate folders](https://github.com/ublue-os/startingpoint/blob/template/scripts/README.md)
```

Let's not add any more "simply" or "easy" loaded terms in the docs.

File: docs/making-your-own.md [hunk 2]
```diff
@@ -43,13 +44,29 @@ You can quickly set up a GitHub repository that builds a functional native conta
...
+git rebase --onto live template
+
+# Perform a force-push to update your "live" branch on GitHub, to deploy.
+# The "lease" ensures that you won't overwrite "live" if GitHub's version
+# is different than your local version (ie. if a team member pushed to it).
+git push --force-with-lease
 ```
 
-If you want to use GitHub through SSH after setting up the repo automatically, run:
+Most of the time, the rebased changes will simply be applied automatically without any need for manual editing. However, if you've modified any core files from the template, then you might have merge conflicts which need to be manually resolved. It's therefore recommended that you use a GUI such as [GitHub Desktop](https://desktop.github.com/) to do your rebase in a visual manner, which will make it very easy to handle any conflicts.
```
Comment (castrojo) [inline docs/making-your-own.md:66]:
```suggestion
Most of the time, the rebased changes will be applied automatically without any need for manual editing. However, if you've modified any core files from the template, then you might have merge conflicts which need to be manually resolved. It's therefore recommended that you use a GUI such as [GitHub Desktop](https://desktop.github.com/) to do your rebase in a visual manner, which handle any conflicts.
```

File: docs/making-your-own.md [hunk 3]
```diff
@@ -43,13 +44,29 @@ You can quickly set up a GitHub repository that builds a functional native conta
...
+
+Note that the `create-ublue-image` sets up the "origin" remote to use HTTPS URLs. If you want to connect to GitHub through SSH instead, simply run the following command to update your local repository's URL:
```
Comment (castrojo) [inline docs/making-your-own.md:68]:
```suggestion
Note that the `create-ublue-image` sets up the "origin" remote to use HTTPS URLs. If you want to connect to GitHub through SSH instead, run the following command to update your local repository's URL:
```

File: docs/making-your-own.md [hunk 4]
```diff
@@ -58,19 +75,20 @@ git remote set-url origin git@github.com:UserName/RepoName.git
...
+    - The `live` branch is the only branch that will be published to your image container registry. However, all branches will be built, which ensures that other branches and pull requests will still be checked for build errors.
+    - You should make the `live` branch your repo's default branch in your GitHub fork's settings.
+    - You should periodically sync changes from `ublue-os/startingpoint:template` into your repo's `template` branch. Then, to get the updates into your customized `live` branch, you can either rebase it on top of `template`, or create a merge-commit with the latest changes from `template`. See the "[automatic setup](#automatic-setup)" section for more information about how to perform the syncing and rebasing of your `live` changes, which is the easiest and cleanest way to update yourself to the latest `template` version, since all of your Git commit history will remain clean and logical.
+4. Change the image name in [the recipe](https://github.com/ublue-os/startingpoint/blob/main/recipe.yml). Simply replace the word `startingpoint` with any custom name of your choice. This is what your image will be called when it's uploaded to your container repository.
```
Comment (castrojo) [inline docs/making-your-own.md:88]:
```suggestion
4. Change the image name in [the recipe](https://github.com/ublue-os/startingpoint/blob/main/recipe.yml). Replace the word `startingpoint` with any custom name of your choice. This is what your image will be called when it's uploaded to your container repository.
```

**Review by castrojo — CHANGES_REQUESTED** [review]

File: docs/making-your-own.md (hunk 1 above)
Comment (Arcitec) [reply docs/making-your-own.md:32]: I was just attempting to follow the existing tone, but agreed! Changed. :)

**Comment by Arcitec** [comment]
@castrojo Ready, everything's been taken care of. :)

**Pull request merged**